LICENSE.md

```diff
@@ -4,7 +4,7 @@ Copyright (c) 2012 Airbnb
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
+in the Software without restriction, including, without limitation, the rights
```

Comment:
these commas are absent in the canonically recognized version of the MIT license, so i think it's best not to add them.

Reply:
I think a comma alone would not likely invalidate the MIT License or change its fundamental nature.

Reply:
I agree, but it might make static license detection difficult.